packages/modals/src/elements/DrawerModal/DrawerModal.tsx

```diff
@@ -22,6 +22,7 @@ import { ThemeContext } from 'styled-components';
 import { useModal } from '@zendeskgarden/container-modal';
 import { useDocument, useText } from '@zendeskgarden/react-theming';
 import { useFocusVisible } from '@zendeskgarden/container-focusvisible';
+import activeElement from 'dom-helpers/activeElement';
```

Comment:
We may need to look into filtering out nested paths in our `external` config for `rollup` since this dependency is being bundled.

Reply:
I would be surprised if rollup doesn't have a mechanism for this already - export paths are a native feature nowadays.